packages/did-auth-siop-op-authenticator/package.json

```diff
@@ -16,16 +16,16 @@
     "@sphereon/did-auth-siop": "0.2.10",
     "@sphereon/pex": "^1.0.2",
     "@sphereon/ssi-sdk-core": "^0.5.0",
-    "@veramo/core": "3.1.1-next.26",
-    "@veramo/utils": "3.1.1-next.26",
+    "@veramo/core": "3.1.2-next.84",
```

Comment:
Is this really necessary. We are upping all our packages and I explicitly stopped at 26, because we needed to do some work to get past that. I doubt that all of a sudden resolved itself

Reply:
Actually Veramo adds partial support to bls keys from version 3.1.2-next.1 on. It would be nice to use the existing implementation.